app/src/forms/form/encryptionKey/routes.js

```diff
@@ -0,0 +1,23 @@
+const routes = require('express').Router();
+const { currentUser, hasFormPermissions } = require('../../auth/middleware/userAccess');
+const validateParameter = require('../../common/middleware/validateParameter');
+const apiAccess = require('../../auth/middleware/apiAccess');
+const rateLimiter = require('../../common/middleware').apiKeyRateLimiter;
```

Comment:
Now that the rateLimiter is applied to the app to do all routes, it's not longer necessary to do it in the routes.js files. However, you are going to get CodeQL warnings that there's no rate limiting (false positive), but FORMS-1303 will fix that.

Reply:
ok, will remove these and update the tests